recipes/navigation.md

```diff
@@ -0,0 +1,136 @@
+import Example from '../components/md/Example';
+
+<Example reactnative>
+
+```js
+import React from 'react';
+import { Button, View, Text, StyleSheet } from 'react-native';
+import { createAppContainer, createStackNavigator } from 'react-navigation';
+
+class FirstScreen extends React.Component {
+  render() {
+    return (
+      <View style={styles.container}>
```

Comment:
I prefer keeping examples readable rather than completely working and with fancy styles
Could you please drop styles related code?

Reply:
i agree :)